rclpy/src/rclpy/_rclpy.c

```diff
@@ -1182,45 +1182,40 @@ rclpy_publish(PyObject * Py_UNUSED(self), PyObject * args)
  *
  * \param[in] period_nsec unsigned PyLong object storing the period of the
  *   timer in nanoseconds in a 64-bit unsigned integer
+ * \param[in] clock pycapsule containing an rcl_clock_t
  * \return a list of the capsule and the memory address
  * \return NULL on failure
  */
 static PyObject *
 rclpy_create_timer(PyObject * Py_UNUSED(self), PyObject * args)
 {
   unsigned PY_LONG_LONG period_nsec;
+  PyObject * pyclock;
 
-  if (!PyArg_ParseTuple(args, "K", &period_nsec)) {
+  if (!PyArg_ParseTuple(args, "KO", &period_nsec, &pyclock)) {
```

Comment:
Should the clock be based first and the period second (as in the other languages)?

Reply:
https://github.com/ros2/rclpy/pull/212/commits/61fac0c9f86212373ee4e2a4426de3fd3da219ae